transfers/transfer.py

```diff
@@ -143,20 +165,28 @@ def get_accession_id(dirname):
         p = subprocess.Popen(
             [script_path, dirname], stdin=subprocess.PIPE,
             stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    except Exception:
-        LOGGER.info('Error when trying to run %s', script_path)
+    # Non-fatal exception, scripts are optional, so warn the user in the logs.
+    except OSError as err:
+        LOGGER.warning('Error when trying to run %s', script_path)
         return None
     output, err = p.communicate()
     if p.returncode != 0:
-        LOGGER.info('Error running %s %s: RC: %s; stdout: %s; stderr: %s',
-                    script_path, dirname, p.returncode, output, err)
+        LOGGER.error('Error running %s %s: RC: %s; stdout: %s; stderr: %s',
+                     script_path, dirname, p.returncode, output, err)
         return None
     output = fsdecode(output)
     try:
         return ast.literal_eval(output)
-    except Exception:
-        LOGGER.info(
-            'Unable to parse output from %s. Output: %s', script_path, output)
+    # Non-fatal exceptions, scripts are optional, so warn the user in the logs.
+    except ValueError as err:
+        LOGGER.warning(
+            "ValueError: Unable to parse output from %s. Output: %s. %s",
+            script_path, output, err)
+        return None
+    except SyntaxError as err:
+        LOGGER.warning(
+            "SyntaxError: Unable to parse output from %s. Output: %s. %s",
+            script_path, output, err)
         return None
```

Comment:
These are so similar that you could just `except (ValueError, SyntaxError) as err: ...`.